CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 1.14.0
+
+This release introduces CoreOS 1151.0.0 in the coreos, and kvm stage1 flavors, and also updates systemd to v231.
```

Comment:
I think this is a bit confusing, what about:
```
This release updates the coreos and kvm flavors, bringing in a newer stable systemd (v231). Several fixes and cgroups-related changes landed in `api-service`, and better heuristics have been introduced to avoid using overlays in non-supported environments. Finally, `run-prepared` now honors options for insecure/privileged pods too.
```